From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Samuel Attard <sattard@salesforce.com>
Date: Fri, 21 Oct 2022 16:29:06 -0700
Subject: build: allow electron to use exec_script

This is similar to the //build usecase so we're OK adding ourselves here

diff --git a/.gn b/.gn
index e6f5cbc9c49f998d63dc8974c5b4789643b2fc87..1df4bbe26fe90eef2dc1a73faaff3c2405993491 100644
--- a/.gn
+++ b/.gn
@@ -169,4 +169,8 @@ exec_script_whitelist =
 
       "//tools/grit/grit_rule.gni",
       "//tools/gritsettings/BUILD.gn",
+
+      "//electron/BUILD.gn",
+      "//third_party/electron_node/deps/base64/BUILD.gn",
+      "//third_party/electron_node/deps/base64/unofficial.gni",
     ]
